project/remote.py

```diff
@@ -73,7 +77,16 @@ def merge_from_upstream_tag():
         actions.append(
             'cd {0} && if [[ -f "{1}{3}.{2}" ]]; then echo "Moving our {1}{3}.{2} back to {1}.{2}";mv "{1}{3}.{2}" "'
             '{1}.{2}";git add {1}.{2};git commit -m "Commiting our {1}.{2}";fi;'
-            .format(self.builddir,self.readMeFileName,self.readMeFileExt,self.readMePSHFile)
+            .format(self.builddir, self.readMeFileName, self.readMeFileExt, self.readMePSHFile)
+        )
+
+        # now if a .github directory was reintroduced from the upstream, delete it, and then if ours is present, rename
+        # it back to .github
+        actions.append(
+            # remove a .github directory that might have been brought in from upstream
+            # @todo should we loop through all possible directories/files that may need to be removed?
+            'cd {0} && [ -d {1} ] && rm -rf {1} && [ -d {2} ] && mv {2} {1}'.format(self.builddir, '.github', 'tmp'
```

Comment:
```suggestion
            'cd {0} && [ -d {1} ] && rm -rf {1} && [ -d {2} ] && mv {2} {1}'.format(self.builddir, '.github', 'tmp.github'
```
?

Reply:
auto PEP8 formatting.  
> E501 - line too long (123 > 120 characters)

I dont have a hard opinion on it.

Reply:
Is the name not incorrect? I thought you had renamed to `tmp.github` not `tmp`?